docs/notes/2.22.x.md

```diff
@@ -109,6 +109,8 @@ Python tools can be [exported from their default bundled lockfiles](https://www.
 New field `pex_build_extra_args` is available on FaaS targets [python_aws_lambda_function](https://www.pantsbuild.org/2.22/reference/targets/python_aws_lambda_function#pex_build_extra_args), 
 [python_aws_lambda_layer](https://www.pantsbuild.org/2.22/reference/targets/python_aws_lambda_layer#pex_build_extra_args), and [python_google_cloud_function]((https://www.pantsbuild.org/2.22/reference/targets/python_aws_lambda_layer#pex_build_extra_args). This allows passing arguments to the `pex` invocation for collecting sources and packaging.
 
+The `runtime` field of [`aws_python_lambda_layer`](https://www.pantsbuild.org/2.22/reference/targets/python_aws_lambda_layer#runtime) or [`aws_python_lambda_function`](https://www.pantsbuild.org/2.22/reference/targets/python_aws_lambda_function#runtime) now has a built-in complete platform configuration for x86-64 Python 3.12. This may allow either deleting a manual `complete_platforms` configuration, or lead to more reliable builds.
```

Comment:
This makes it sound like you can either delete the manual config or have more reliable builds (but not both...)

Plus, it's unclear in what way the builds would otherwise be unreliable.

Maybe rephrase to say that this provides stable support for 3.12 lambdas out of the box, allowing deleting manual configuration?

Reply:
Ah, yes, good catch!

```suggestion
The `runtime` field of [`aws_python_lambda_layer`](https://www.pantsbuild.org/2.22/reference/targets/python_aws_lambda_layer#runtime) or [`aws_python_lambda_function`](https://www.pantsbuild.org/2.22/reference/targets/python_aws_lambda_function#runtime) now has a built-in complete platform configuration for x86-64 Python 3.12. This provides stable support for Python 3.12 lambdas out of the box, allowing deleting manual `complete_platforms` configuration if any.
```

Reply:
LGTM